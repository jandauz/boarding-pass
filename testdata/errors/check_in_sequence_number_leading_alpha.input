M1DESMARAIS/LUC       EABC123 YULFRAAC 0834 326J001AA025 100