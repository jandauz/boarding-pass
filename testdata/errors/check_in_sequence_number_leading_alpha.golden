bcbp: Invalid data format:
  boarding pass data:
  | "M1DESMARAIS/LUC       EABC123 YULFRAAC 0834 326J001AA025 100"
  |                                                      ^ got "A025 "
  |
  = reason: data for "Check-in Sequence Number" must be 4 digits with leading zeroes followed by an optional alpha or whitespace
